typescript/cli/src/deploy/warp.ts

```diff
@@ -243,13 +244,16 @@ async function resolveWarpIsmAndHook(
 ): Promise<WarpRouteDeployConfig> {
   return promiseObjAll(
     objMap(warpConfig, async (chain, config) => {
-      const chainAddresses = await context.registry.getChainAddresses(chain);
+      const registryAddresses = await context.registry.getAddresses();
+      const ccipContractCache = new CCIPContractCache(registryAddresses);
```

Comment:
We did obviously discuss this, but still is tough to read. We'll have to come back and clean this up later